helm/charts/hpe-csi-driver/README.md

```diff
@@ -70,7 +70,27 @@ helm install my-hpe-csi-driver hpe-storage/hpe-csi-driver -n hpe-storage -f myva
 
 Due to the [helm limitation](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations) to not support upgrade of CRDs between different chart versions, helm chart upgrade is not supported.
 
-Our recommendation is to uninstall the existing chart and install the chart with the desired version. CRDs will be preserved between uninstall and install.
+For the upgradtion of chart from 2.0.0 to 2.1.0 we need to apply Primera3PAR metata CRDs before the uninstallation of driver as mentioned in steps below:
+```
+kubectl apply -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/helm/charts/hpe-csi-driver/crds/hpevolumeinfos_v2_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/helm/charts/hpe-csi-driver/crds/hpevolumegroupinfos_v2_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/helm/charts/hpe-csi-driver/crds/snapshotgroupinfos_v2_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/helm/charts/hpe-csi-driver/crds/hpereplicated_deviceinfo_v2_crd.yaml
+```
+For the upgradtion of the chart from 2.0.0 to 2.1.0 with replication information follow the below steps before the uninstalltion of the driver.  
+```
+Create the deployment using the below command, which will modify the rcg-info record to the new key RCGCreatedByCSP.
+kubectl create -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/yaml/csi-driver/primera3par/convert-rcg-info.yaml
```

Comment:
This needs to be rewritten as a Job controller: https://kubernetes.io/docs/concepts/workloads/controllers/job/

Reply:
We also need to remove the infinity sleep from the rcg-info binary and rebuild the image.

Reply:
Moved manifest to its own release directory: https://raw.githubusercontent.com/hpe-storage/co-deployments/master/yaml/rcg-info/v1.0.0/convert-rcg-info.yaml

Reply:
> We also need to remove the infinity sleep from the rcg-info binary and rebuild the image.

Addressed  this in PR 
https://github.hpe.com/hpe/hpe_3par_primera_rcginfo/pull/2